### PR TITLE
fix: fix tengine vs2019 build for lib and examples

### PR DESCRIPTION
**Pull request by zchrissirhcz**
环境 VS2019
修复 tengine lib 编译时，类型报错，  void* / float* / uint8_t* 显示转换
修复 很多 example 中的 std::max / std::min 编译报错

### CODE REVIEW

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -101,6 +101,7 @@ ENDIF()
 
 # add examples with opencv
 # set(OpenCV_DIR /mnt/d/ubuntu/opencv_install/linux-armv7/lib/cmake/opencv4)
+set(OpenCV_DIR "E:/lib/opencv/4.5.2" CACHE PATH "")
```
Comment (kalcohol) [inline examples/CMakeLists.txt]: remove this line

**Review by kalcohol — DISMISSED** [review]

File: examples/CMakeLists.txt (hunk 1 above)
Comment (zchrissirhcz) [reply examples/CMakeLists.txt]: OK, removed in next commit.

**Comment by zchrissirhcz** [comment]
Also tested on local ubuntu 20.04 machine.
Examples built OK.

**Review by BUG1989 — APPROVED** [review]
lgtm!

**Review by BUG1989 — APPROVED** [review]
lgtm!

File: examples/CMakeLists.txt (hunk 1 above)
Comment (BUG1989) [reply examples/CMakeLists.txt]: done.

**Comment by cmdbug** [comment]
zz yes!

**Comment by BUG1989** [comment]
@cmdbug Who are you ?

**Comment by cmdbug** [comment]
> @cmdbug Who are you ?

小莫愁

**Pull request merged**